v2.0/admin-ui-time-series.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Admin UI Time Series Graphs
+toc: false
+---
+
+<div id="toc"></div>
+
+The Admin UI displays time series graphs of key metrics. Time series graphs are useful to visualize and monitor data trends. You can hover over each graph to see actual point-in-time values.
+
+<img src="{{ 'images/admin_ui_hovering.gif' | relative_url }}" alt="CockroachDB Admin UI" style="border:1px solid #eee;max-width:100%" />
+
+{{site.data.alerts.callout_info}}By default, CockroachDB stores timeseries metrics for the last 30 days, but you can reduce the interval for timeseries storage. Alternately, if you are exclusively using a third-party tool such as <a href="monitor-cockroachdb-with-prometheus.html">Prometheus</a> for timeseries monitoring, you can disable timeseries storage entirely. For more details, see this <a href="operational-faqs.html#can-i-reduce-or-disable-the-storage-of-timeseries-data-new-in-v2-0">FAQ</a>.
```

Comment:
it might be clear already, but it might be useful to be explicit here that doing disabling storage will make time series charts not work (including the sparklines on the clusterviz)